data-governance/no-code-access.mdx

```diff
@@ -40,11 +40,12 @@ which are exposed to your users.
 
 ## One IP address
 
-If you need to set your firewall to allow our agent to exercise your web app, please use this IP address: `35.192.162.70`
+If you need to set your firewall to allow our agent to exercise your web app, please use this IP address: `35.192.162.70` or
+`34.159.153.198` (europe proxy).
 
 <Warning>
-  Please let us know so that we can set your organisation to always use our
-  proxy. We do not have this option as self-service yet. Pleae use this email
+  Please let us know so that we can set your organization to always use our
+  proxy. We do not have this option as self-service yet. Please use our email
   address: contact@octomind.dev
 </Warning>
```

Comment:
not true anymore, all test targets use the proxy currently -> remove this section / warning please

Reply:
ah, I didn't know!

Reply:
> not true anymore, all test targets use the proxy currently -> remove this section / warning please

I'll do another commit